backend/endpoints/rooms.py

```diff
@@ -41,6 +41,16 @@ async def create_room(
 
     DB = DataStorage(org_id)
     room_obj = Room(**request.dict(), org_id=org_id, created_by=member_id)
+
+    if (
```

Comment:
Do we need parentheses here? @o4codes

Reply:
@zxenonx 
It's been removed now.
Thanks